components/SocialSharingVertical.vue

```diff
@@ -32,7 +32,7 @@
     </div>
     <div class="share-icon">
       <a
-        :href="'https://twitter.com/share?text=' + quote + '&url=' + organisation.site_address + url"
+        :href="'https://twitter.com/share?text=' + quote + '&url=' + encodeURI(organisation.site_address + url)"
```

Comment:
@monarkatfactly I think you should form the URL completely first and then encode it completely. I think there will be an issue in this case if the 'quote' is in another language. Please look into this once.

Reply:
With telugu language it's working perfectly, problem was with only URL